src/reducers/versions.spec.tsx

```diff
@@ -179,6 +189,25 @@ describe(__filename, () => {
     });
   });
 
+  describe('getVersionFiles', () => {
+    it('returns all the files for a given version', () => {
```

Comment:
Should we store multiple files for a version in order to be able to assert that this does in fact return *all files* for a version?